### PR TITLE
add chef development kit

### PR DESCRIPTION
**Pull request by raisercostin**
Chef Development Kit for Windows 10 and Windows 2012r2

### CODE REVIEW

File: chefdk.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "env_add_path": "bin",
+    "## psmodule": {
+        "name": "chef"
+    },
+    "checkver": {
+        "url": "https://downloads.chef.io/chefdk/stable",
+        "re": "Chef Development Kit\\D*([\\d.]+)"
+    },
+    "autoupdate": {
+        "note": "Thanks for using autoupdate, please test your updates!",
```
Comment (Ash258) [inline chefdk.json]: Remove note

File: chefdk.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        "CHEFDK_ENV_FIX": "1"
+    },
+    "installer": {
+        "script": [
+            "extract_7zip \"$dir\\chefdk.zip\" \"$dir\"",
+            "Remove-Item \"$dir\\chefdk.zip\""
+        ]
+    },
+    "env_add_path": "bin",
+    "## psmodule": {
```
Comment (Ash258) [inline chefdk.json]: Why `##`?

**Comment by Ash258** [comment]
Close this PR or remove chefdk.json from #1439

File: chefdk.json (hunk 2 above)
Comment (raisercostin) [reply chefdk.json]: removed. was a temporary comment. sorry

**Pull request closed without merging**